src/api.js

```diff
@@ -37,4 +37,11 @@ export const postComment = (review_id, username, body) => {
     .then((response) => {
         return response.data.comment
     })
+}
+
+export const fetchCategories = () => {
+    return reviewApi.get("/categories")
+    .then((response) => {
+        return response.data.categories
+    })
```

Comment:
Here in this file you can reuse the function that fetches all the reviews, but add 'axios params' to that request. For this see NC Notes: Axios/Axios Params.